keel-core/src/main/kotlin/com/netflix/spinnaker/keel/constraints/DependsOnConstraintEvaluator.kt

```diff
@@ -38,6 +40,27 @@ class DependsOnConstraintEvaluator(
       artifact,
       version,
       requiredEnvironment.name
+    ) && allVerificationsSucceededIn(
```

Comment:
It's funny that this question _just_ came up in our all hands meeting and @robfletcher was saying that verification would not necessarily block promotion of the artifact to the next environment, but I think this makes total sense. My recollection is that we had agreed on this behavior, right?

Reply:
Yeah probably. I wasn't sure even as I was saying that.